### ISSUE TITLE
Norton 360 and Windows Defender Smart Scan says malicious

### ISSUE THREAD

**Issue by BobRichmond** [Healix/Gw2Launcher#242]
Just a heads up for others trying to download the 7075 build. Norton 360 and Windows (10) Defender Smart Scan both said this build was malicious for me. You might have to make exceptions in both programs to stop it from being deleted. You also should restart the system after making the exceptions to get them to work.

**Comment by Healix** [comment]
New releases will always have this problem. Gw2Launcher isn't paid for, so I'm in the slow lane - it won't be "safe" until enough people use it.

**Comment by BobRichmond** [comment]
Yes I realized that before I posted. However there may be other new users that don't and haven't figured out how to mitigate it before the programs view it as ok.
